src/components/workspace/create-verification-modal/verification-stepper/stepper-body/stepper-body-item/stepper-body-item.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, StepLabel, Typography, StepContent } from '@mui/material';
+
+import { useStyles } from './styles';
+
+interface IStepperBodyProps {
+  step: { label: string; description: string };
+  labelProps: { optional?: React.ReactNode; error?: boolean };
+  activeStep: number;
+  thirdStep: boolean;
+  index: number;
+}
+
+export function StepperBodyItem({
+  step,
+  labelProps,
+  activeStep,
+  thirdStep,
+  index,
+}: IStepperBodyProps): React.ReactNode {
+  const { classes } = useStyles();
+
+  return (
+    <Box>
+      <StepLabel {...labelProps} className={classes.textItemTitle}>
+        {step.label}
+        <Typography
+          className={
+            thirdStep === false && activeStep === 3 && index === 2
```

Comment:
this can be moved to the separate function and then called here, also think about using enums for the steps